.github/dependabot.yml

```diff
@@ -5,7 +5,7 @@
 
 version: 2
 updates:
-  - package-ecosystem: "gomod" # See documentation for possible values
+  - package-ecosystem: "github-actions" # See documentation for possible values
```

Comment:
I think we now have 2 github actions. I think we still want to know about critical updates/vulns in go packages, even if we cannot immediately update with dependabot.

We also have a NPM package in this repo, which we should probably remove before merging repos.